test/functional/roles_test.js

```diff
@@ -0,0 +1,29 @@
+Feature('Admin Web - Roles');
+
+Scenario('add all the roles', I => {
+  I.amOnPage(`https://admin-web-div-ccd-definitions-pr-${process.env.CHANGE_ID}.service.core-compute-preview.internal`);
```

Comment:
We're only running these test in preview (PRs) so this URL should be fine for now..